tokenserver/views.py

```diff
@@ -96,6 +96,13 @@ def valid_assertion(request):
             raise _unauthorized("invalid-timestamp")
         raise _unauthorized("invalid-credentials")
 
+    try:
+        tokenVerified = assertion['idpClaims']['fxa-tokenVerified']
+    except KeyError:
+        tokenVerified = True
```

Comment:
nit: `assertion['idpClaims'].get('fxa-tokenVerified', True)` will have the same effect, and save you the try-except